modules/claims_api/app/controllers/concerns/claims_api/poa_verification.rb

```diff
@@ -81,11 +81,20 @@ def verify_representative_and_veteran(logged_in_representative_user, target_vete
       end
 
       def poa_code_in_organization?(poa_code)
+        organization = ::Veteran::Service::Organization.find_by(poa: poa_code)
         representative = ::Veteran::Service::Representative.where('? = ANY(poa_codes)', poa_code).first
-        raise 'Power of Attorney not found' if representative.blank?
+
+        organization.present? && representative.user_types.include?('veteran_service_officer')
+      end
+
+      def poa_code_is_individual?(poa_code)
```

Comment:
it felt weird to not have this method as a contrast to `poa_code_in_organization?`, feels more logically readable in instances where we would call it.

Also, the logic is subtlety different between the 2. For individuals, we just need to check it _doesn_'t have a matching org but _does_ have a matching individual. By contrast, for orgs, we need the org code to exist _and_ have an individual with said code.

This check is more future proofed against anyone who might be both individually accredited _and_ acting for  an org. While that doesn't seem to occur, given the existing OGC, I think it's possible, and it's also how Tamara Ellis and John Doe are constructed.